src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -29,9 +33,22 @@ const char* uct_rdmacm_cm_ep_str(uct_rdmacm_cm_ep_t *cep, char *str,
         NULL
     };
 
+    if (ucs_sockaddr_is_known_af(local_addr)) {
+        ucs_sockaddr_str(local_addr, local_ip_port_str, UCS_SOCKADDR_STRING_LEN);
+    } else {
+        ucs_strncpy_safe(local_ip_port_str, "<not set>", UCS_SOCKADDR_STRING_LEN);
```

Comment:
maybe `<invalid>` or `<invalid address>` or just `?`

Reply:
i think 'invalid' implies some error. the intention here was to handle a case where the address was not set yet (but it will be later on after the rdmacm_id will be bound to an address). wdyt?

Reply:
how can we tell the difference between a case where it wasn't set at all, and a case where it's set to some invalid value?

Reply:
we can't by the message itself. but i think we can tell by the place it was called from - where the error message is printed from. we can tell from there if the address should have been set or not.

Reply:
IMO `<invalid>` is better, be cause it reflects the current state. `<not set>` makes an assumption that the reason the address is invalid is because it was not set yet, and this assumption is not always true.

Reply:
ok